planner/Readme

```diff
@@ -1,30 +0,0 @@
-Usage
-------
-Run the planner service by
-1. configuring the path to the FF planner
-2. calling "python PlannerService.py"
-
-Service entry-points and example test files using Curl
---------------------
-Test : GET HOST:port/planner/test
-Example : curltest-test
-
-upload a domain file : POST HOST:port/planner/upload_domain
-Example : curltest-upload-domain
-
-upload a problem file : POST HOST:port/planner/upload_problem
-Example : curltest-upload-problem
-
-generate a plan : GET HOST:port/planner/generate_plan
-Example : curltest-plan : supplies a json file containing planner, domain and problem
-specification
-Example : curltest-jsonplan : supplies a json string containing planner, domain and problem
```

Comment:
Does the new service have all this info in its README?

Also, for the future, it's better to split different kinds of changes into multiple branches so that we can review and merge them separately.